### PR TITLE
updated the error title and error subtitle

### PR DESCRIPTION
**Pull request by anubhav1206**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: static/images/briefcase.png
Comment (Cahllagerfeld) [inline static/images/briefcase.png:1]: I think this icon isn't used

File: src/lib/components/header.svelte [hunk 1]
```diff
@@ -27,15 +27,21 @@
...
       alt="hubber"
       aria-label="hubber"
     />
   </a>
+  <img
+    src="/images/message-code.png"
+    class="w-12 justify-center object-contain md:w-14"
+    alt="message-code"
+    aria-label="message-code"
+  />
```
Comment (Cahllagerfeld) [inline src/lib/components/header.svelte:44]: What is this change for?

File: package-lock.json [hunk 2]
```diff
@@ -1189,25 +1189,25 @@
       }
     },
     "node_modules/@sveltejs/kit": {
-      "version": "1.0.11",
-      "resolved": "https://registry.npmjs.org/@sveltejs/kit/-/kit-1.0.11.tgz",
-      "integrity": "sha512-FcbR2/jlKtSfFBXwnqgpWnoOG54foLQB2esdkkdz7uei0+KnK5ulWZw0lJbMzXTy7kfxRvH1spxlXoBpPizUqg==",
+      "version": "1.15.9",
+      "resolved": "https://registry.npmjs.org/@sveltejs/kit/-/kit-1.15.9.tgz",
+      "integrity": "sha512-Og+4WlguPVPS0PmAHefp4KxvTVZfyDN09aORVXIdKSzqzodSJiLs7Fhi/Q0z0YjmcoNLWF24tI0a6mTusL6Yfg==",
```
Comment (Cahllagerfeld) [inline package-lock.json:1194]: When upgrading the dependencies here, did you do compatibility checks?

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
Could you please open an issue for discussion around this before creating the PR.

File: package-lock.json (hunk 2 above)
Comment (anubhav1206) [reply package-lock.json:1194]: I did not do the checks.

File: src/lib/components/header.svelte (hunk 1 above)
Comment (anubhav1206) [reply src/lib/components/header.svelte:44]: This code block is for the icon.

File: static/images/briefcase.png
Comment (anubhav1206) [reply static/images/briefcase.png:1]: Sure i'll open an issue on this.

**Comment by Cahllagerfeld** [comment]
closing this for now due to inactivity

**Pull request closed without merging**